### PR TITLE
Include Ansible Installation method in docs

### PR DESCRIPTION
**Pull request by armab**
We just released massive update for [`ansible-st2`](https://github.com/StackStorm/ansible-st2) `v0.6.0` https://github.com/StackStorm/ansible-st2/releases which introduces full platform support (4 platforms) and includes a lot of testing, bug-fixing and polishing.

With this update I'm confident about the quality, additionally changes are tested against [4 platforms in Travis](https://travis-ci.org/StackStorm/ansible-st2).

I think it worth to include Ansible as one of the installation methods now :100:

### CODE REVIEW

File: docs/source/install/ansible.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+---------------------------
+.. sourcecode:: bash
+
+    git clone https://github.com/StackStorm/ansible-st2.git
+    cd ansible-st2
+
+    ansible-playbook stackstorm.yml
+
+
+Please refer to https://github.com/StackStorm/ansible-st2 for updates and more detailed examples, descriptions and code.
```
Comment (Mierdin) [inline docs/source/install/ansible.rst]: Should this be a link too? Not sure how RTD will render this

Comment (LindsayHill) [inline docs/source/install/ansible.rst]: Bare links get rendered as a link - see https://docs.stackstorm.com/mistral_yaql.html?highlight=yaql#testing-yaql-expressions for an example. In the RST it appears as http://yaqluator.com/., and it gets rendered as a link

**Comment by LindsayHill** [comment]
I'm not sure about putting it as a top-level menu item yet. It needs to have references from some other sections, e.g. /install/index.html. 

We also need to clean up the orphan file here https://github.com/StackStorm/st2docs/blob/master/docs/source/install/puppet_chef_salt_ansible.rst

File: docs/source/install/index.rst [hunk 2]
```diff
@@ -54,6 +54,7 @@ Once it completes successfully, you will see the following output:
     Ubuntu 14.04 / 16.04 <deb>
     RHEL 7 / CentOS 7 <rhel7>
     RHEL 6 / CentOS 6 <rhel6>
+    Ansible Playbooks <ansible>
```
Comment (armab) [inline docs/source/install/index.rst]: @LindsayHill Sub-menu under "Installation" Is close to previous structure, see: https://docs.stackstorm.com/1.3/install/index.html
I'm for keeping it visible to get more users trying + testing this method for us, what could be especially beneficial considering we're running Ansible for own infra.

Comment (armab) [inline docs/source/install/index.rst]: Alternative approach is to polish and revert entire 'puppet_chef_salt_ansible.rst' page what possibly can encourage contributions for other installation methods. In a meantime I left that page if we reconsider in some future time. WDYT?

Comment (LindsayHill) [inline docs/source/install/index.rst]: Right now, there's not really enough content here to justify having a dedicated Ansible page. Either we put this content in another page (e.g. refer to it from the main install page), or we overhaul the older doc, push Ansible as the 'preferred/supported' way of doing it, with notes that older Puppet/Chef stuff exists

Comment (armab) [inline docs/source/install/index.rst]: My initial idea was to keep it short as possible and refer to GitHub repo.

But I can bring enough content from the `ansible-st2` README, - it's not a problem. Including more detailed installation examples, requirements and configuration vars.

Comment (LindsayHill) [inline docs/source/install/index.rst]: Yes, I think that bringing in some content from the README would give it enough 'meat'

Comment (armab) [inline docs/source/install/index.rst]: Added more content like descriptions, details and examples.

**Pull request merged**